File: 10 - Python/Exercícios/03 - Estruturas Repetitivas/For/Exercício 06.py
#Exercício - Média Ponderada

n = int(input("Digite quantos casos você vai digitar: "))

for i in range(n):

    print("Digite três números:")
    a = float(input(""))
    b = float(input(""))
    c = float(input(""))

    media = (a * 2 + b * 3 + c * 5) / 10
    print("Média =", round(media,1))

File: 06 - C/Exercícios/03 - Estruturas Repetitivas/For/Exercício 06.c
//Exercício - Média Ponderada

#include <stdio.h>

int main(){

    int i, N;
    double a, b, c, media;

    printf("Quantos casos voce vai digitar? ");
    scanf("%d", &N);

    for(i = 1; i <= N; i++){

        printf("Digite tres numeros:\n");
        scanf("%lf", &a);
        scanf("%lf", &b);
        scanf("%lf", &c);

        media = (a * 2 + b * 3 + c * 5) / 10.0;

        printf("Media = %.1lf\n", media);
    }

    return 0;
    
}